impl/src/main/java/org/ehcache/impl/serialization/CompactJavaSerializer.java

```diff
@@ -52,7 +53,7 @@
 
   private final AtomicInteger nextStreamIndex = new AtomicInteger(0);
```

Comment:
Actually, will open an issue and fix elsewhere since it needs to be fixed on 3.0 as well.